src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const total = 40 * days;
+
+  if (days >= 3 && days < 7) {
```

Comment:
OK, but you can avoid combining conditions in the task if you put if-statements in the proper order